doc/model/axes/axis-cartesian-discrete.xml

```diff
@@ -32,6 +32,25 @@
             For additional information, please see <c:link to="pvc.options.axes.CartesianAxis" />.
         </c:documentation>
 
+        <c:property name="labelRotationDirection" 
```

Comment:
These two options also need to be included in a separate complex type, `AnyNonHierarchicalCartesianAxis`. To be able to reuse these in both types, these should be moved into a facet type, for example: `FlattenedDiscreteCartesianAxisFacet`. Then this facet is referenced by both of the concrete complex types (in their `facets` attribute): `FlattenedDiscreteCartesianAxis` and `AnyNonHierarchicalCartesianAxis`.